src/D3D12MemAlloc.cpp

```diff
@@ -2528,7 +2528,11 @@ class IntrusiveLinkedList
     IntrusiveLinkedList(IntrusiveLinkedList&& src);
     IntrusiveLinkedList& operator=(const IntrusiveLinkedList&) = delete;
     IntrusiveLinkedList& operator=(IntrusiveLinkedList&& src);
+#ifdef _DEBUG

```

Comment:
I think this is not right. The user of the library can define his own `D3D12MA_HEAVY_ASSERT` which is not necessarily empty/non-empty depending on `_DEBUG` macro.